buildpack.toml

```
@@ -13,3 +13,8 @@ id = "*"
 
 [[buildpack.licenses]]
 type = "BSD-3-Clause"
+
+[metadata]
+[metadata.release]
+[metadata.release.image]
+repository = "docker.io/heroku/procfile-cnb"
```

(ref https://github.com/heroku/buildpacks-python/pull/58#discussion_r1282002498)

```suggestion
[metadata.release]
image = { repository = "docker.io/heroku/procfile-cnb" }
```